csm_web/scheduler/views/section.py

```diff
@@ -119,6 +119,43 @@ def create(self, request):
         serializer = self.serializer_class(section)
         return Response(serializer.data, status=status.HTTP_201_CREATED)
 
+    def destroy(self, request, pk=None):
+        """
+        Handle request to delete section through the UI;
+        deletes mentor and spacetimes along with it
+        """
+        section = get_object_or_error(self.get_queryset(), pk=pk)
```

Comment:
`self.get_queryset()` only gets unbanned sections; just in case something went wrong and the coord is banned from the section, maybe use `Courses.objects.all()`.